bot/exts/fun/uwu.py

```diff
@@ -116,10 +61,57 @@ class Uwu(Cog):
     def __init__(self, bot: Bot):
         self.bot = bot
 
+    def _word_replace(self, input_string: str) -> str:
+        """Replaces words that are keys in the word replacement hash to the values specified."""
+        for word in WORD_REPLACE_HASH:
+            input_string = input_string.replace(word, WORD_REPLACE_HASH[word])
+        return input_string
+
+    def _char_replace(self, input_string: str) -> str:
+        """Replaces characters with 'w'."""
```

Comment:
It doesn't replace all characters so we can clarify that like this:
```suggestion
        """Replace certain characters with 'w'."""
```
Do feel free to improve it in other ways.